notebooks/official/model_evaluation/automl_tabular_regression_model_evaluation.ipynb

```diff
@@ -76,12 +76,11 @@
         "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.    ! pip3 install --upgrade google-cloud-aiplatform google-cloud-pipeline-components kfp matplotlib {USER_FLAG} -q
</pre><p>and new lines:</p><pre class="ql-syntax" spellcheck="false">! pip3 install --upgrade --quiet {USER_FLAG} google-cloud-aiplatform \
                                             google-cloud-pipeline-components \
                                             kfp \
                                             matplotlib 
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1179/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>done</p>